src/Blaster.WebApi/Features/Shared/feature-flag.js

```diff
@@ -10,7 +10,7 @@ export default class FeatureFlag {
     }
 
     flagExists(name) {
-        return this.flags[name] !== null;
+        return this.flags[name] !== undefined;
```

Comment:
(not a js expert) but i believe its more idiomatic in js to use `if name in this.flags`?

Reply:
Changed it.